samples/msgext-ai-sentiment-analysis/csharp/MEAISentimentAnalysis/Pages/SentimentModel.cshtml

```diff
@@ -0,0 +1,48 @@
+﻿@page
+@model MEAISentimentAnalysis.Pages.SentimentModel
+@{
+}
+<head>
+    <script src="https://res.cdn.office.net/teams-js/2.14.0/js/MicrosoftTeams.min.js"
```

Comment:
User Latest SDK version, Instead of 2.14.0

Reply:
Updated